### DISCUSSION TITLE
How to add a simplified C2H4/Air reaction mechanism?

### DISCUSSION THREAD

**Discussion by Pandakii** [AMReX-Combustion/PeleC#837] [Q&A] [answered]
I want to use a simplified C2H4/Air reaction mechanism to simulate a C2H4 jet flame. How can I add the simplified reaction mechanism shown below into PeleC? I have examined the existing files containing detailed reaction mechanism, but I am uncertain about how to simplify it.
![1724675448145](https://github.com/user-attachments/assets/908b0714-0656-4a4b-bf38-78bb373b64df)

**Comment by marchdf** [comment, marked as answer]
Hi, if you have the corresponding chemkin file, then you could convert that for PeleC: https://amrex-combustion.github.io/PelePhysics/Ceptr.html. Or you could write the yaml file manually since it is pretty human-friendly. Maybe starting with something like: https://github.com/AMReX-Combustion/PelePhysics/blob/development/Mechanisms/CH4_lean/mechanism.yaml. Or you could write it up in cantera and have that write out the yaml file.

**Reply by Pandakii** [reply]
Thanks for your help. The simplified reaction mechanism works now.
